backend/dataall/modules/shares_base/services/sharing_service.py

```diff
@@ -112,26 +112,26 @@ def approve_share(cls, engine: Engine, share_uri: str) -> bool:
                             if not success:
                                 share_successful = False
                         except Exception as e:
-                            log.error(f'Error occurred during sharing of {type.value}: {e}')
+                            log.exception(f'Error occurred during sharing of {type.value}')
                             ShareStatusRepository.update_share_item_status_batch(
                                 session,
                                 share_uri,
                                 old_status=ShareItemStatus.Share_Approved.value,
                                 new_status=ShareItemStatus.Share_Failed.value,
-                                share_item_type=processor.type.value,
+                                share_item_type=processor.type,
                             )
                             ShareStatusRepository.update_share_item_status_batch(
                                 session,
                                 share_uri,
                                 old_status=ShareItemStatus.Share_In_Progress.value,
                                 new_status=ShareItemStatus.Share_Failed.value,
-                                share_item_type=processor.type.value,
+                                share_item_type=processor.type,
                             )
                             share_successful = False
                 return share_successful
 
             except Exception as e:
-                log.error(f'Error occurred during share approval: {e}')
+                log.exception('Error occurred during share approval')
```

Comment:
Why are we removing the logging of the exception message? is it included in log.exception?

Reply:
Yes log.exception is printing both the stack trace and the message